app/src/main/contrib/components/Responsibilities/ModelRunParameters/ModelRunParameterSetsList.tsx

```diff
@@ -27,19 +28,39 @@ export class ModelRunParameterSetsListComponent extends RemoteContentComponent<P
         }
     }
 
+
+    private makeDownloadableSignatureLinkContent(set: ModelRunParameterSet) {
+        const setData = {
+            id: set.id,
+            disease: set.disease,
+            uploaded_by: set.uploaded_by,
+            uploaded_on: set.uploaded_on
+        };
+        const signature = Base64.encode(JSON.stringify(setData));
+        const signatureData = [setData, { signature }];
+        return "data: text/json;charset=utf-8," + encodeURIComponent(JSON.stringify(signatureData, null, 2));
+    }
+
     private renderTable(props: Props) {
         if (props.sets && props.sets.length > 0) {
             const items = props.sets.map((set, i) => <tr key={i}>
                 <td>{set.description}</td>
                 <td>{set.uploaded_by}</td>
                 <td>{longTimestamp(new Date(set.uploaded_on))}</td>
+                <td>
+                    <a
+                        href={this.makeDownloadableSignatureLinkContent(set)}
+                        download={`signature${set.id}.json`}
```

Comment:
Lets remove this `.json` file extension (and have it as a file with no extension)